BundIDTests/ControllerCallbackTests.swift

```diff
@@ -43,25 +37,31 @@ class MockActivationResult: NSObject, ActivationResultProtocol {
 }
 
 final class ControllerCallbackTests: XCTestCase {
-
+    private var cancellables = Set<AnyCancellable>()
+    
+    override func tearDown() async throws {
+        cancellables.removeAll()
+    }
+    
     func testOKActivationCode() throws {
         
         let activationResult = MockActivationResult(code: .OK)
-            
+        
         let controllerCallback = ControllerCallback()
-            
+        
         let completionExpectation = expectation(description: "Expect complection")
         let valueExpectation = expectation(description: "Expect value")
-        let cancellable = controllerCallback.publisher.sink { completion in
+        controllerCallback.publisher.sink { completion in
             completionExpectation.fulfill()
             XCTAssertEqual(completion, .finished)
         } receiveValue: { value in
             valueExpectation.fulfill()
             XCTAssertEqual(value, .processCompletedSuccessfullyWithoutRedirect)
         }
+        .store(in: &cancellables)
```

Comment:
storing the cancellable resolves the unused warning I got for the `cancellable` variable